pkg/security/probe/field_handlers.go

```diff
@@ -442,3 +442,49 @@ func (fh *FieldHandlers) ResolveEventTimestamp(ev *model.Event) time.Time {
 	}
 	return ev.Timestamp
 }
+
+// ResolvePackageName resolves the name of the package providing this file
+func (fh *FieldHandlers) ResolvePackageName(ev *model.Event, f *model.FileEvent) string {
+	if f.PkgName == "" {
+		if pkg := fh.resolvers.SBOMResolver.ResolvePackage(ev.ProcessCacheEntry.ContainerID, f); pkg != nil {
+			f.PkgName = pkg.Name
+			f.PkgVersion = pkg.Version
+		}
+	}
+	return f.PkgName
+}
+
+// ResolvePackageVersion resolves the version of the package providing this file
+func (fh *FieldHandlers) ResolvePackageVersion(ev *model.Event, f *model.FileEvent) string {
+	if f.PkgVersion == "" {
```

Comment:
Should we ensure that the path was fully resolved before resolving the package ?